auxiliary.py

```diff
@@ -0,0 +1,30 @@
+import os
+from matplotlib.cbook import get_sample_data
+import matplotlib.patches as mpatches
+import matplotlib.pyplot as plt
+
+
+uco = mpatches.Patch(color=(1., 1., 0.), label='UCO')
+buffer = mpatches.Patch(color=(1., 0.5, 0.), label='Buffer')
+pyc = mpatches.Patch(color=(1., 0., 0.), label='PyC')
+sic = mpatches.Patch(color=(0.6, 0.3, 0.), label='SiC')
+matrix = mpatches.Patch(color=(0.63, 0.63, 0.63), label='Matrix')
+block = mpatches.Patch(color=(0.61, 1., 0.91), label='Block')
```

Comment:
consider a more descriptive name. Is it a fuel block, a moderator graphite block?